### PR TITLE
Converted GB Camera sections 2.2.6 - 3.1.2 

### PR DESCRIPTION
**Pull request by ApianbelleDev**
I'm nowhere close to done, but I wanted to at least open a PR with what I _do_ have, as to hopefully get feedback and merge with what's already there. as for the entire thing, please be patient as I'm working as quickly as I can, and I'd appreciate feedback with what I can do better.

### CODE REVIEW

File: src/Gameboy_Camera.md [hunk 1]
```diff
@@ -171,6 +171,97 @@ Those registers form a 4×4 matrix with 3 bytes per element. They handle ditheri
   <figcaption>Edge extraction.</figcaption>
 </figure> 
 
+## Game Boy Camera Timings
+The capture process is started when the A000 register of the Game Boy Camera cartridge is written with any value with bit 0 set to "1".
+The Game Boy Camera cartridge is one of the few cartridges that use the PHI signal (clock from the GB). That signal is a 1MHz clock (1047567 Hz). the M6438FP chip needs a clock input too, which is half the frequency of the PHI pin (0.5Mhz, 524288Hz). The reason for that is that the sensor chip sometimes handles hte signals on the rising edge of the clock, but other times on the falling edge.
```
Comment (avivace) [inline src/Gameboy_Camera.md:176]:
```suggestion
The Game Boy Camera cartridge is one of the few cartridges that use the PHI signal (clock from the GB). That signal is a 1MHz clock (1047567 Hz). the M6438FP chip needs a clock input too, which is half the frequency of the PHI pin (0.5Mhz, 524288Hz). The reason for that is that the sensor chip sometimes handles the signals on the rising edge of the clock, but other times on the falling edge.
```

File: src/Gameboy_Camera.md [hunk 2]
```diff
@@ -171,6 +171,97 @@ Those registers form a 4×4 matrix with 3 bytes per element. They handle ditheri
...
+
+The clock signal during read perioud must be the same as the one used during the exposure time, but if the clock during the read period is too slow, the sensor will continue increasing the charg values of each pixel so the image will appear to be taken with a higher exposure time. The brightness doesn't always seem to increase, however. there appears to be some kind of limit.
+
+## The Game Boy Camera sensor (M64282FP)
+the M64282FP does some processing to the captured image. First, it performs an edge control, then it does gain controll, and lastly, it deas level control. the resulting analog value is the one that can be read in the Vout pin. The sensor can capture infrared radiation, so the images can be a bit strange compared to others captured by better sensors.
```
Comment (ISSOtm) [inline src/Gameboy_Camera.md:214]:
```suggestion

The M64282FP does some processing to the captured image.
First, it performs an edge control, then it does gain control, and lastly, it does level control.
The resulting analog value is the one that can be read in the V<sub>out</sub> pin.
The sensor can capture infrared radiation, so the images can be a bit strange compared to others captured by better sensors.

```

File: src/Gameboy_Camera.md [hunk 3]
```diff
@@ -171,6 +171,97 @@ Those registers form a 4×4 matrix with 3 bytes per element. They handle ditheri
...
+## The M64282FP registers
+
+### Register 1
+This corresponds to the register A001 of the Game Boy Camera. When shooting, the values change based on how much light there is.
```
Comment (ISSOtm) [inline src/Gameboy_Camera.md:218]:
```suggestion

This corresponds to the register $A001 of the Game Boy Camera.
When shooting, the values change based on how much light there is.
```

File: src/Gameboy_Camera.md [hunk 4]
```diff
@@ -171,6 +171,97 @@ Those registers form a 4×4 matrix with 3 bytes per element. They handle ditheri
...
+
+| Symbol      | Bits | Operation                                         |
+| ----------- | ---- | ------------------------------------------------- |
+| N           | 7    | Exclusively set vertical edge enhancement mode.   |
+| VH          | 5-6  | Select vertical/horizontal edge enhancement mode. |  
+| G           | 0-4  | Analog output gain.                               |
+
+<break><break>
+<break><break>
+
```
Comment (ISSOtm) [inline src/Gameboy_Camera.md:228]:
```suggestion
```

File: src/Gameboy_Camera.md [hunk 5]
```diff
@@ -171,6 +171,97 @@ Those registers form a 4×4 matrix with 3 bytes per element. They handle ditheri
...
+| 1  | 0  | 0  | 0  | 26.0 |
+| 1  | 0  | 0  | 1  | 29.0 |
+| 1  | 0  | 1  | 0  | 32.0 |
+| 1  | 0  | 1  | 1  | 35.0 |
+| 1  | 1  | 0  | 0  | 38.0 |
+| 1  | 1  | 0  | 1  | 41.0 |
+| 1  | 1  | 1  | 0  | 45.5 |
+| 1  | 1  | 1  | 1  | 51.5 |
+
+If G4 = "1", the total gain is the previous one plus 6dB. The Game Boy Camera uses 00h, 04h, 08h, and 0Ah at 14.0dB, 20.0dB, 26.0dB, and 32dB respectively, which translate to a gain of 5.01, 10.00, 19.95, and 39.81. The Game Boy Camera seems to like to duplicate the game in each step.
```
Comment (ISSOtm) [inline src/Gameboy_Camera.md:248]:
```suggestion
If G4 = "1", the total gain is the previous one plus 6 dB.
The Game Boy Camera uses $00, $04, $08, and $0A at 14 dB, 20 dB, 26 dB, and 32 dB respectively, which translate to a gain of 5.01, 10.00, 19.95, and 39.81.
The Game Boy Camera seems to like to duplicate the game in each step.

```

File: src/Gameboy_Camera.md [hunk 6]
```diff
@@ -171,6 +171,97 @@ Those registers form a 4×4 matrix with 3 bytes per element. They handle ditheri
...
+## Registers 2 and 3
+Registers 2 and 3 contain the exposure time(a 16 bit unsigned value). According to the M64282FP datasheet, each step is 16 µs. the GB needs 16 PHI clocks for every step. However, if N = "1", exposure_steps should be greater than or equal to 0030h.
```
Comment (ISSOtm) [inline src/Gameboy_Camera.md:250]:
```suggestion

Registers 2 and 3 contain the exposure time (a 16 bit unsigned value).
According to the M64282FP datasheet, each step is 16 µs.
The GB needs 16 PHI clocks for every step.
However, if N = "1", `exposure_steps` should be greater than or equal to $0030.
```

File: src/Gameboy_Camera.md [hunk 7]
```diff
@@ -171,6 +171,97 @@ Those registers form a 4×4 matrix with 3 bytes per element. They handle ditheri
...
+```
+    u16 exposure_steps ((Reg2)<<8) | [Reg3]
+    Step time = 1 / 1048576 Hz * 16 = 0,954 µs* 16 = 15,259 µs
```
Comment (ISSOtm) [inline src/Gameboy_Camera.md:253]:
```suggestion
u16 exposure_steps ((Reg2) << 8) | [Reg3]
Step time = 1 / 1048576 Hz * 16 = 0,954 µs* 16 = 15,259 µs
```
I'm not entirely sure about the parens vs. brackets on the two registers?

File: src/Gameboy_Camera.md [hunk 8]
```diff
@@ -171,6 +171,97 @@ Those registers form a 4×4 matrix with 3 bytes per element. They handle ditheri
...
+``` 
+It's a bit less than the 16 µs the datasheet says, but it's close enough. below are some example values to get acceptable pictures under various light conditions
```
Comment (ISSOtm) [inline src/Gameboy_Camera.md:255]:
```suggestion

It's a bit less than the 16 µs the datasheet says, but it's close enough. 
Below are some example values to get acceptable pictures under various light conditions:
```

File: src/Gameboy_Camera.md [hunk 9]
```diff
@@ -171,6 +171,97 @@ Those registers form a 4×4 matrix with 3 bytes per element. They handle ditheri
...
+
+| Value | Conditions                                            |
+| ----- | ----------------------------------------------------- |
+| 0030h | Objects under direct sunlight.                        |
+| 0300h | Objects not under direct sunlight.                    |
+| 0800h | Room during the day with good light.                  |
+| 2C00h | Room at night with no light.                          |
+| 5000h | Room at night with no light, only a reading lamp .    |
+| F000h | Room at night with only a TV on in the background.    |
```
Comment (ISSOtm) [inline src/Gameboy_Camera.md:264]:
```suggestion
| $0030 | Objects under direct sunlight.                        |
| $0300 | Objects not under direct sunlight.                    |
| $0800 | Room during the day with good light.                  |
| $2C00 | Room at night with no light.                          |
| $5000 | Room at night with no light, only a reading lamp .    |
| $F000 | Room at night with only a TV on in the background.    |
```

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Here are a bunch of editorial tweaks. Thank you for taking the time!

**Comment by avivace** [comment]
Replaced by https://github.com/gbdev/pandocs/pull/478

**Pull request closed without merging**